bin/test-workflow/8_app_verify_authentication.sh

```diff
@@ -88,6 +90,12 @@ if [[ "$PLATFORM" == "openshift" ]]; then
     SIDECAR_PORT_FORWARD_PID=$!
   fi
 
+  if [[ " ${install_apps[*]} " =~ " summon-sidecar-jwt " ]]; then
+    sidecar_jwt_pod=$(get_pod_name test-app-summon-sidecar-jwt)
+    oc port-forward "sidecar_jwt_pod" 8081:8080 > /dev/null 2>&1 &
```

Comment:
I think this should be \`oc port\-forward "$sidecar\_jwt\_pod"\`.
Also line 96 should be `SIDECAR_JWT_PORT_FORWARD_PID`